### PR TITLE
adding new files

### PR DESCRIPTION
**Pull request by antuanettaV**

### CODE REVIEW

**Comment by bhas** [comment]
Hi Hanna, I think something went wrong. Is this your React 1 Week 1 homework? I don't see any files related to the homework. :(

**Comment by antuanettaV** [comment]
@bhas  Dear Bjørn, hello. Thank you for your message. I have updated the PR, please kindly check it. The files OurCrew.js, OurPartners.js and OurValues.js are in the components folder. Sorry for inconveniences and thank you once again.

File: app.js
Comment (vdaguenet) [inline app.js:1]: I don't think this file is part of the React homework

File: app/about_us/page.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
+import styles from './page.module.css'; 
+
+// OurValues 
+const OurValues = () => {
```
Comment (vdaguenet) [inline app/about_us/page.js:4]: You also have this component in `components/OurValues.js`. You need to decide on which one you want to keep.

File: app/about_us/page.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+      <p className="number">03</p>
+      <h3 className="value-title">Be Better. Always push forward. We're on a tireless pursuit to become better and expect those around us to do the same.</h3>
+      <p className="number">04</p>
+      <h3 className="value-title">Be a Professional. There's important work to be done. Speak up. Be present. Provide value, with honesty and candor.</h3>
+    </div>
+  );
+};
+
+// OurCrew 
+const OurCrew = () => {
```
Comment (vdaguenet) [inline app/about_us/page.js:25]: Same comment as the component above

File: app/about_us/page.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+            <p className="job-title">{member.title}</p>
+          </div>
+        ))}
+      </div>
+    </div>
+  );
+};
+
+// OurPartners 
+const OurPartners = () => {
```
Comment (vdaguenet) [inline app/about_us/page.js:72]: Same comment as the component above

File: app/about_us/page.js [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+        <p className="brand-name">CBC</p>
+        <p className="brand-name">Queen's</p>
+        <p className="brand-name">Samsung</p>
+      </div>
+    </div>
+  );
+};
+
+// Crew 
+export const Crew = () => {
```
Comment (vdaguenet) [inline app/about_us/page.js:89]: I think this is `AboutUs`

File: app/layout.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+      <body className={inter.className}>
+        <Navbar />
+        {children}
+        {/* TASK - React 1 week 1 */}
+        {/* Import and use the Footer component here */}
+        {/* Footer found in the ui/Footer.js folder */}
+        <OurValues />
+        <OurPartners />
+        <OurCrew />
+         <Footer />
```
Comment (vdaguenet) [inline app/layout.js:29]: Be careful with indentation

**Review by vdaguenet — COMMENTED** [review]
The project structure is very hard to follow. Did you fork the project as explained in the installation guide https://github.com/HackYourFuture-CPH/react-1-hw?tab=readme-ov-file#installation?
